### PR TITLE
BXMSPROD-583 cleanWS removed and DEFAULT_BRANCH job parameter adapted

### PR DESCRIPTION
**Pull request by Ginxo**
BXMSPROD-583

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -16,8 +16,6 @@ pipeline {
         stage ("Clean") {
             steps {
                 echo "[INFO] Cleaning the workspace."
```
Comment (winklerm) [inline Jenkinsfile.nightly]: Wondering if this echo should be there?

Comment (mareknovotny) [reply Jenkinsfile.nightly]: well, the whole stage Clean is then useless, why not remove it at all?

Comment (Ginxo) [reply Jenkinsfile.nightly]: removed

**Comment by Ginxo** [comment]
Since we have transformed the multibranch pipeline to pipeline with parameters because of QE requirements the pipeline should be adapted.

**Pull request merged**